ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -41,16 +42,16 @@ export function WorkflowsRow(props: WorkflowsRowProps) {
                     />
                     <PhaseIcon value={wf.status.phase} />
                 </div>
+                <div className='columns small-2'>
+                    <a className='overlay' href={workflowLink}></a>
+                    <WorkflowsRowName metadata={wf.metadata} />
+                </div>
                 <Link
                     to={{
                         pathname: uiUrl(`workflows/${wf.metadata.namespace}/${wf.metadata.name}`),
                         search: `?uid=${wf.metadata.uid}`
                     }}
-                    className='small-11 row'>
-                    <div className='columns small-2'>
-                        {(wf.metadata.annotations && wf.metadata.annotations[ANNOTATION_TITLE]) || wf.metadata.name}
-                        {wf.metadata.annotations && wf.metadata.annotations[ANNOTATION_DESCRIPTION] ? <p>{wf.metadata.annotations[ANNOTATION_DESCRIPTION]}</p> : null}
-                    </div>
+                    className='small-8 row'>
```

Comment:
```suggestion
                    className='small-9 row'>
```

this should be 9 columns, not 8. This is currently causing the UI to look pretty janky: 
![Screenshot 2024-01-22 at 1 05 49 AM](https://github.com/argoproj/argo-workflows/assets/4970083/31cfe434-8ed4-4388-97dd-fcf76a8c0ecb)

Reply:
that doesn't fully fix the jankiness, something else is up with the CSS here

Reply:
something about the new columns is causing these columns not to flex all the way, but removing them solves all of it.

this issue also breaks a few other things, like the details drawer (if you can get it to open, usually I end up at the details page trying to click) is now missing the columns on the left as well:
![Screenshot 2024-01-28 at 12 21 52 PM](https://github.com/argoproj/argo-workflows/assets/4970083/c4acac46-7468-41f4-ba15-e57d25059ab6)

and my overflow fix from #11908 was re-broken since the column is smaller:
![Screenshot 2024-01-28 at 12 22 46 PM](https://github.com/argoproj/argo-workflows/assets/4970083/dfa88567-05b0-481b-b3d6-148734cbdd57)

I'll have a PR for this out shortly, just noting some of my observations / the bugs that I encountered fixed

Reply:
Fixed all of the above in #12580